tools/refguide_check.py

```diff
@@ -60,7 +60,7 @@
 BASE_MODULE = "numpy"
 
 PUBLIC_SUBMODULES = [
-    'core',
+    '_core',
```

Comment:
Not public anymore. That said there's lots of public types with docstrings defined inside of `_core` so maybe it's correct to leave this here? Someone with more experience with this file should weigh in.

Reply:
Right, this one should be removed.